about.html

```diff
@@ -0,0 +1,170 @@
+                </li>
+                <li>
+                    <a href="">My Page</a>
+                </li>
+                <li>
+                    <a href="">Logout</a>
+                </li>
+            </ul>
+        </div>
+        <nav class="bottom-navbar">
+            <div class="logo">
+                <a class="logoa" href="index.html">
+                    <span>Zi</span>
+                    RO
+                    <br>
+                    <SPan>Music</SPan>
+                </a>
+            </div>
+            <ul>
+                <li>
+                    <a href="index.html">Home</a>
+                </li>
+                <li>
+                    <a href="about.html">About</a>
+                </li>
+                <li>
+                    <a href="#join">Join</a>
+                </li>
+                <li>
+                    <a href="#partners">Sponsor</a>
+                </li>
+                <li>
+                    <a href="">News</a>
+                </li>
+                <li>
+                    <a href="">Campaign</a>
+                </li>
+            </ul>
+        </nav>
+    </header>
+
+    <main>
+        <section class="main">
+            <nav class="Mobile-navbar">
+                <img id="cancelBtn" src="images/Icon - Cancel.svg" alt="">
+                <i id="hamburger" class="fas fa-bars hamburger" aria-hidden="true"></i>
+                <ul id="hamburger-list">
+                    <li>
+                        <a href="index.html">Home</a>
+                    </li>
+                    <li>
+                        <a href="about.html">About</a>
+                    </li>
+                    <li>
+                        <a href="">Join</a>
+                    </li>
+                    <li>
+                        <a href="">Sponsor</a>
+                    </li>
+                    <li>
+                        <a href="">News</a>
+                    </li>
+                    <li>
+                        <a href="">Campaign</a>
+                    </li>
+                </ul>
+            </nav>
+            <div class="container">
+                <div class="content">
+                    <p class="content-hellow">"Hellow! Music World"</p>
+                    <h1 class="main-content-heading">ZIRO MUSIC FESTIVAL NIGHT 2023</h1>
+                    <p class="content-detail">A Ziro Music Festival is a live music performance in front of an audience. 
+                        The performance may be by a single musician, sometimes then called a recital, 
+                        or by a musical ensemble, such as an orchestra, choir, or band. 
+                    </p>
+                    <p class="content-lastline">Please contact us for any further questions <br> 
+                        about zero Music Festival 2023! <br>
+                        <strong>anshumannegi777@gmail.com</strong>
+                    </p>
+                </div>
+        </div>
```

Comment:
Kindly use the same font-family, color, and font-size and weight as in the provided design.

- [ ] Also your background image is not properly aligned as in the provided template. It is all distorted and is hindering the visibility of text in a certain place around your email. Kindly fix this issue.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://github.com/anshuman7negi/ConcertPage/assets/33268927/833aa960-30ad-4ec4-861a-4f7d8a3559ba) | ![image](https://github.com/anshuman7negi/ConcertPage/assets/33268927/7256f712-9c09-4395-bd92-986d6b9564c0) |

Reply:
Fixed.